openapi/types.rs

```diff
@@ -0,0 +1,61 @@
+/// Types used for REST communication with the server
+// mod status;
+// mod error;
+// mod warning;
+
+// use status::Status;
+// use error::Errors;
+// use warning::Warnings;
+
+use serde::{Deserialize, Serialize};
+use utoipa::{IntoParams, ToSchema};
+use uuid::Uuid;
+
+/// Reduced telemetry from the asset
+/// Reduced bandwidth for nominal flight
+#[allow(dead_code)]
+#[derive(Debug, Copy, Clone, IntoParams, ToSchema)]
+#[derive(Deserialize, Serialize)]
+pub struct BasicTelemetryData {
+
+    /// Asset ID
+    pub uuid: Uuid, // 128 bits, can we shrink this?
+
+    /// Current Position
+    pub position: PositionData,
+
+    // Active Status
+    // pub status: Status,
+
+    // // Active Warnings
+    // pub warnings: Warnings,
+
+    // // Active Errors
+    // pub errors: Errors,
+
+    /// Data Checksum
+    checksum: u32
+}
+
+impl BasicTelemetryData {
+    #[allow(dead_code)]
+    fn checksum_update(&mut self) {
+        self.checksum = 0x0;
+        todo!();
+    }
+}
+
+/// Global location of the asset
+#[allow(dead_code)]
+#[derive(Debug, Clone, IntoParams, ToSchema, Copy)]
+#[derive(Deserialize, Serialize)]
+pub struct PositionData {
+    /// current latitude
+    pub latitude: f32,
+
+    /// current longitude
+    pub longitude: f32,
+
+    /// current altitude in meters
+    pub altitude_meters: f32,
```

Comment:
Might use the OrderedFloat type?

Reply:
Fixed: https://github.com/Arrow-air/svc-telemetry/pull/1/commits/f74db5a95f5d7a1c81b5aaaf0fd208f6ce09c63b